.github/workflows/kind-e2e-upgrade.yaml

```diff
@@ -116,6 +121,13 @@ jobs:
         # Create a cluster!
         kind create cluster --config kind.yaml
 
+    - name: Deploy certificates for upstream traffic
+      if: matrix.upstream-traffic == 'tls'
```

Comment:
I guess you run this at some point before commenting out the strategy property above (havent checked all the commits). Correct?

Reply:
This is upgrade action. As we cannot enable TLS on previous release (which does not implement this...), we cannot enable it. 
Another action (kind-e2e) enables this and runs.

Reply:
Ah, I guess you misunderstood about my typo :sweat_smile: I fixed it. (kind-e2e also did not run with TLS unexpectedly but I fixed it.)